CONTRIBUTING.md

```diff
@@ -19,6 +19,34 @@ The following is a git shell example for installing the app on your local machin
 cd /web_container/
 git clone https://github.com/CenterForDigitalHumanities/deer.git deer
 ```
+### Tomcat installation Guide (Windows)
+
+Note:The Standard Java Development Kit (JDK) needs to be installed for this to work.
+
+Many of the developers use Tomcat9 as their preferred webserver. To avoid combatability issues it's reccomended that new developers use the same or something compatable.
+
+Download Tomcat9 installation wizard from the above link and set up following the installer instrctions. Many people find it usefull to download into the root of a drive or other easily-pathed location.
+
+In powershell or command prompt navigate to the "bin" folder of the tomcat files and run the startup.bat file. This can be done using the following example of pathing commands.
+
+```
+E:\>cd Tomcat9
+
+E:\Tomcat9>cd bin
+
+E:\Tomcat9\bin>startup.bat
+```
+The user then can navigate to the "webapps" folder of the tomcat files. Here a new folder for the DEER project can be created. Navigating to this file in gitbash or windows command prompt is the proper location to clone the git files using the above git shell example.
+
+Note: The name of the folder that the code is cloned into is case sensitive and will become the address of the webserver's localhost address (naming this folder deer will mean that the address will be localhost:####/deer. Naming this folder ROOT or replacing the existing ROOT folder will allow you to navigate to the page without a folder name at the end of the address)
```

Comment:
This is out of order.  This and the paragraph above should happen before they start tomcat.  In general, this guidance is unclear.  Perhaps offer the example where you install it in `E:\Tomcat9\webapps\`